README.md

```diff
@@ -81,21 +83,199 @@ sudo acs run
 sudo acs run --quiet
 ```
 
-### Get
+<br>
+
+## Get
+
+### Flags
+`--raw` is the only used flag for the `get` command.
+
+### Subcommands
```

Comment:
hmm maybe it’s better to put detailed instructions in another markdown file instead of the readme, keeping it relatively short

Reply:
Yea, thats the idea with the hide away stuff, but maybe another md doc would be good